Assets/Scripts/DataManagerScript.cs

```diff
@@ -18,9 +19,15 @@ public class DataManagerScript : MonoBehaviour {
 	public static bool CRTMode = true;
 	public static bool isChallengeMode = false;
 
+	// Flag for login on initial launch
+
+	public static bool shouldActivateMenu = false;
+
 	// UI Flags
 
-	public static int playerControllingMenus = 1;
+	public static int gamepadControllingMenus;
+	public static User userControllingMenus;
```

Comment:
Ooh, what's a User object?

Reply:
It's the bare bones of an Xbox Live profile! It's from the DLL via `using Users;`. It's basically just Gamertag, UID, local ID on the system, and... one more thing I can't recall.